src/Microsoft.AspNet.Mvc.ModelBinding/Metadata/DefaultMetadataDetails.cs

```diff
@@ -7,21 +7,19 @@
 namespace Microsoft.AspNet.Mvc.ModelBinding.Metadata
 {
     /// <summary>
-    /// A cache of metadata objects for a <see cref="DefaultModelMetadata"/>.
+    /// Holds associated metadata objects for a <see cref="DefaultModelMetadata"/>.
     /// </summary>
     /// <remarks>
-    /// These instances are shared by all <see cref="DefaultModelMetadata"/> instances representing
-    /// the same <see cref="Type"/>, property, or parameter. Any modifications to the data must be
-    /// thread-safe for multiple readers and writers.
+    /// Any modifications to the data must be thread-safe for multiple readers and writers.
     /// </remarks>
-    public class DefaultMetadataDetailsCache
+    public class DefaultMetadataDetails
     {
         /// <summary>
-        /// Creates a new <see cref="DefaultMetadataDetailsCache"/>.
+        /// Creates a new <see cref="DefaultMetadataDetails"/>.
         /// </summary>
         /// <param name="key">The <see cref="ModelMetadataIdentity"/>.</param>
         /// <param name="attributes">The set of model attributes.</param>
-        public DefaultMetadataDetailsCache(ModelMetadataIdentity key, IReadOnlyList<object> attributes)
+        public DefaultMetadataDetails(ModelMetadataIdentity key, IReadOnlyList<object> attributes)
         {
             Key = key;
             Attributes = attributes;
```

Comment:
old nit: `BindingMetadata` and two properties after it lack periods in their summaries